server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/EmailSubmissionSet.scala

```diff
@@ -28,15 +29,22 @@ import org.apache.james.core.MailAddress
 import org.apache.james.jmap.core.Id.{Id, IdConstraint}
 import org.apache.james.jmap.core.Properties.toProperties
 import org.apache.james.jmap.core.SetError.SetErrorDescription
-import org.apache.james.jmap.core.{AccountId, Id, SetError, UuidState}
+import org.apache.james.jmap.core.{AccountId, Id, SetError, UTCDate, UuidState}
+import org.apache.james.jmap.mail.EmailSubmissionId.DATE
 import org.apache.james.jmap.method.{EmailSubmissionCreationParseException, WithAccountId}
 import org.apache.james.mailbox.model.MessageId
 import play.api.libs.json.JsObject
 
 object EmailSubmissionId {
   def generate: EmailSubmissionId = EmailSubmissionId(Id.validate(UUID.randomUUID().toString).toOption.get)
+
+  val DATE: Instant = Instant.parse("2023-04-14T10:00:00.00Z")
```

Comment:
hardcode "2023-04-14T10:00:00.00Z" in production?
I feel a bit incorrect here 
or it is a right spec?